README.md

```diff
@@ -8,15 +8,15 @@
 [![documentation_status](https://readthedocs.org/projects/bentoml/badge/?version=latest)](https://docs.bentoml.org/)
 [![join_slack](https://badgen.net/badge/Join/BentoML%20Slack/cyan?icon=slack)](https://join.slack.bentoml.org)
 
-BentoML let you create machine learning powered prediction service in minutes and bridges the gap between data science and DevOps.
+BentoML let you create a machine learning powered prediction service in minutes and bridges the gap between data science and DevOps.
```

Comment:
Should we do something like this?
```suggestion
BentoML enables users to create a machine learning powered prediction service in minutes and bridges the gap between data science and DevOps.
```

Reply:
I think "let's you" is fine, FWIW.

Maybe something like "BentoML is a tool that allows the creation of..." would be more in line with standard wording.

Reply:
Yea, I'm also in favor of using the 2nd person in general